src/main/java/io/aiven/kafka/connect/transforms/ConcatFieldsConfig.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2019 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.transforms;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+
+final class ConcatFieldsConfig extends AbstractConfig {
+    public static final String FIELD_NAMES_CONFIG = "field.names";
+    private static final String FIELD_NAMES_DOC =
+        "A comma separated list of fields to concatenate.";
+    public static final String FIELD_NAME_CONFIG = "field.name";
```

Comment:
This config name should be a bit more distinctive than `field.names`. Maybe something like `output.field.name` or `result.field.name`.

Reply:
Yeah I really like `output.field.name` I'll go that route. 

- [ ] rename `field.names` to `output.field.name`